v3/src/components/graph/models/graph-model.ts

```diff
@@ -102,5 +130,19 @@ export function isSetAttributeIDAction(action: ISerializedActionCall): action is
   return action.name === "setAttributeID"
 }
 
+export interface SetGraphVisualPropsAction extends ISerializedActionCall {
+  name: "setGraphVisualProps"
+  args: [GraphAttrRole, string]
```

Comment:
```suggestion
  args: [string | number | boolean]
```
All of the actions in question take a single argument of one of these types.